src/applications/mhv-landing-page/containers/App.jsx

```diff
@@ -76,11 +76,11 @@ const App = () => {
         const unreadMessages = countUnreadMessages(folders);
         setUnreadMessageCount(unreadMessages);
       }
-      if (enabled && hasMHVAccount) {
+      if (loggedIn && hasMHVAccount) {
         loadMessages();
       }
     },
-    [enabled, hasMHVAccount],
+    [loggedIn, hasMHVAccount],
```

Comment:
Do you really need loggedIn here? I feel just the hasMHVAccount check would be enough with it checking if the user is logged in there as well. I feel it is confusing when reading the logic to have loggedIn here.

Reply:
I'm wondering about this as well. It seems like `selectHasMHVAccountState` should return `false` if a user is _not_ logged in, because `state?.user?.profile?.mhvAccountState` should evaluate to undefined.

I also wonder how many times this "effect" will get called when a user is not logged in, given `RequiredLoginView` should   trigger the redirect upon initial inspection of `user.login.currentlyLoggedIn`

Reply:
The enabled selector reduced to checking if the user was logged in. Seems like this could be reduced further. Nice.

Reply:
Latest push removes isLoggedIn selector from the App.jsx component.